### PR TITLE
speeding up container build times and locking version numbers (closes #38)

### LINKED ISSUE #38: Update Pipfile with static version numbers for dependencies
# Summary

It is best to have version numbers than just "*"

# Details

Need to find what versions are currently out there for our dependencies, and lock our Pipfile on then

# Outcome

We should have static version numbers for all dependencies

### PR DESCRIPTION
**Pull request by nlMeminger**
Closes #38.

This PR makes a change to Dockerfile that will allow docker to cache changes better and speed up the time it takes to build a container significantly.  This PR also locks the dependencies under the packages section on current versions for each package found in pypi.python.org

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -3,14 +3,17 @@
 FROM python:3.6-stretch
 
 WORKDIR /app
-COPY . /app
+COPY Pipfile /app
```
Comment (justwheel) [inline Dockerfile:6]: In this case, the `Pipfile` does not need to be committed separately. The `Pipfile.lock` is used for installing dependencies in the `pipenv install` step.

File: Pipfile [hunk 2]
```diff
@@ -4,15 +4,15 @@ verify_ssl = true
...
-pyyaml = "*"
+django = "==2.2.6"
+"argon2-cffi" = "==19.2.0"
+django-compressor = "==2.3"
+"psycopg2-binary" = "==2.8.4"
+bcrypt = "==3.1.7"
+mysqlclient = "==1.4.4"
+django-haystack = "==2.8.1"
+whoosh = "==2.7.4"
+pyyaml = "==5.1.2"
```
Comment (justwheel) [inline Pipfile:15]: Awesome! This will close #38.

**Review by justwheel — CHANGES_REQUESTED** [review]
Hey @nlMeminger, this looks great. A clever trick to speed up the build time. :smile: 

I only left one suggestion, the `Pipfile` does not need to be copied separately, only the `Pipfile.lock`. Otherwise it's ready to go!

File: Dockerfile (hunk 1 above)
Comment (nlMeminger) [reply Dockerfile:6]: So removing that line will make the pipenv install command on line 13 fail with this error: 

ERROR: Pipfile not found! Aborting! Please ensure that the file exists and is located in your project root directory.

So copying the Pipfile over with the lock file is required.

**Review by justwheel — APPROVED** [review]
Thansk for clarifying. LGTM! Merging. :ocean:

**Pull request merged**